.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -1,23 +1,28 @@
 name: 🐛 | Bug Report
-description: >
-  File a bug report
+description: Submit a bug report
 title: "[🐛 | Bug]: "
 labels:
   - bug
   - triage
 body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for helping improve Parsec! Please fill in the required fields below and add any extra
+        information that can help us understand the issue.
   - type: input
     id: version
     attributes:
-      label: Version
-      description: What version of our software are you running ?
-      placeholder: version
+      label: Parsec version tested on:
+      description: You can find it in Parsec GUI title bar (or by running `parsec.cli --version` in a terminal).
```

Comment:
`parsec --version` also work and seems more natural for me since `parsec.cli` is mostly for admin operations

Reply:
Does not work for me :(
```bash
$ parsec --version
Usage: python -m parsec.cli core gui [OPTIONS] [URL]
Try 'python -m parsec.cli core gui --help' for help.

Error: No such option: --version

$ parsec.cli --version
parsec, version v2.15.0
```

Reply:
It work on `2.16` :thinking:

Reply:
A lot of cool stuff in `2.16`, we should release it! :grinning: